docs/run-platform-server/running.mdx

```diff
@@ -13,36 +13,68 @@ If you chose containerized installation, you don't execute the horizon binary di
+  Now deploy horizon onto the cluster using the helm chart:
 
-    <CodeExample>
+  <CodeExample>
 
   ```bash
-  helm install myhorizon_role stellar/horizon \
+  helm install my-horizon stellar/horizon \
   --namespace my-horizon-namespace-on-cluster \
-  -f values.yaml \
-  --devel
+  --set ingest.persistence.enabled=true \
+  --set web.enabled=true \
+  --set web.existingSecret=my-db-secret \
+  --set global.image.horizon.tag=2.26.1 \
+  --set global.network=testnet \
+  --set ingest.existingSecret=my-db-secret \
+  --set ingest.horizonConfig.captiveCoreUseDb=true \
+  --set ingest.resources.limits.cpu=1 \
+  --set ingest.resources.limits.memory=6Gi
   ```
 
+  </CodeExample>
+
+  This example of helm chart usage, highlights some key aspects:
+
+  - uses the `global.network=[testnet|pubnet]` parameter, this automates generation of all the horizon configuration parameters specific to network such as archive urls, captive core config, etc and other parameters as mentioned on [Configuring](./configuring.mdx).
+  - enables all roles on the deployment instance: ingesting, api, transaction submission.
+  - to customize further, the best approach is to download the [horizon helm chart values.yaml](https://github.com/stellar/helm-charts/blob/main/charts/horizon/values.yaml), update the settings in your local copy of values.yaml, and pass to helm install, rather than have many individual `--set` on helm install:
+
+    <CodeExample>
+
+    ```bash
+    helm install myhorizon stellar/horizon \
+    --namespace my-horizon-namespace-on-cluster \
+    --values values.yaml
+    ```
+
     </CodeExample>
 
+  - specific to customizing network configuration parameters, if you want to connect to a network other than presets of `testnet` or `pubnet`, then you won't use `global.network`, instead, use local copy of [values.yaml](https://github.com/stellar/helm-charts/blob/main/charts/horizon/values.yaml) and set `ingest.coreConfig`, and refer to [\_core-config.tpl](https://github.com/stellar/helm-charts/blob/main/charts/horizon/templates/_core-config.tpl) for example of all the key/value pairs to include.
+
+  - minimum resource limits, verify whether `LimitRange` defaults are defined on the target namespace in kubernetes for deployment, if so, ensure that the defaults provide at least minimum resource limits of `6Gi` of memory and `1` cpu. Otherwise, define the limits explicitly on the helm install via the `ingest.resources.limits.*` shown in example, to ensure the deployed pods have adequate resources.
```

Comment:
Is the "minimum resource limits" sort of a heading here? If yes, maybe we can write it as - "Minimum Resource Limits:"

Reply:
was treating it as more of a detail related to parent section, following the existing pattern of a bullet list for related info, rather than pulling up to section header. the compute resources will be initially stated in prerequisites.mdx, which is active now - https://github.com/stellar/go/issues/4960